app/views/admin/enterprises/form/_business_details.html.haml

```diff
@@ -31,24 +31,42 @@
     = f.label :invoice_text, t('.invoice_text')
   .omega.eight.columns
     = f.text_area :invoice_text, style: "width: 100%; height: 100px;"
-
-.row
+.row{ data: { controller: 'terms-and-conditions', "terms-and-conditions-message-value": t('js.admin.enterprises.form.images.immediate_terms_and_conditions_removal_warning') } }
```

Comment:
Now there's only one `terms-and-conditions` controller which also encapsulate the modal content.